nexus-repository-r/src/test/java/org/sonatype/nexus/repository/r/internal/AssetKindTest.java

```diff
@@ -22,14 +22,13 @@
 import static org.sonatype.nexus.repository.cache.CacheControllerHolder.CONTENT;
 import static org.sonatype.nexus.repository.cache.CacheControllerHolder.METADATA;
 import static org.sonatype.nexus.repository.r.internal.AssetKind.ARCHIVE;
-import static org.sonatype.nexus.repository.r.internal.AssetKind.PACKAGES;
 
 public class AssetKindTest
     extends TestSupport
 {
   @Test
   public void cacheTypes() throws Exception {
     assertThat(ARCHIVE.getCacheType(), is(equalTo(CONTENT)));
-    assertThat(PACKAGES.getCacheType(), is(equalTo(METADATA)));
+    assertThat(AssetKind.METADATA.getCacheType(), is(equalTo(METADATA)));
```

Comment:
import AssetKind.METADATA

Reply:
Done in f5a0c4341c6ed3aa9669cb5cd973d162f75a6acd